.github/workflows/install_and_test.yml

```diff
@@ -66,3 +66,12 @@ jobs:
           hacspec-halo2-fstar
           hacspec-weierstrass-coq
           hacspec-weierstrass-fstar
+
+    - name: Push to Cachix
+      if: ${{ github.event_name == 'workflow_dispatch'  || github.event_name == 'merge_group'  }}
+      run: |
+        nix profile install nixpkgs#cachix nixpkgs#jq
+        nix build --json \
+          | jq -r '.[].outputs | to_entries[].value' \
+          | cachix push hax
```

Comment:
So this doesn't need any sort of authentication?

Reply:
It does, but I added a secret on the repo's config!